### PR TITLE
v1.4.2 - Change the appearance of the applet icon when colors are not…

### PR DESCRIPTION
**Pull request by claudiux**
… in use, to better alert the user when the sound is greater than 100%.

Hi,

Can you merge this sound150 branch with your master branch, please ?

Best regards.

Claudiux

### CODE REVIEW

**Comment by germanfr** [comment]
Hi, @claudiux. The license of your applet is missing in the `info.json` file. Please, can you add it in this or another PR?
Here is a reference on how it should look like: https://github.com/linuxmint/cinnamon-spices-applets/search?l=JSON&q=license

**Comment by claudiux** [comment]
Hi @germanfr. I have added the licence.

**Comment by claudiux** [comment]
Thank you @jaszhix !

**Comment by claudiux** [comment]
In the https://cinnamon-spices.linuxmint.com/applets/view/306 page the "Last Edition" field remains at "1 month". It will be changed soon?

File: sound150@claudiux/files/sound150@claudiux/metadata.json [hunk 1]
```diff
@@ -14,5 +14,5 @@
         "3.6",
         "3.8"
     ],
-    "last-edited": 1523640550
+    "last-edited": 1526065577
```
Comment (jaszhix) [inline sound150@claudiux/files/sound150@claudiux/metadata.json]: Maybe try removing this from your metadata.json? Not sure why the site won't update, but Cinnamon is supposed to handle the `last-edited` field internally.

**Pull request merged**